For_university/experiment1.py

```diff
@@ -0,0 +1,72 @@
+import random
+
+SIZE_OF_UNIVERSE = 100
+NUMBER_OF_SUBSETS = 40
+CARDINALITY = 4
+REMAINING_SUBSET = NUMBER_OF_SUBSETS - CARDINALITY
+ELEMENTS_OF_UNIVERSE = list(range(SIZE_OF_UNIVERSE))
+RATIO = int(NUMBER_OF_SUBSETS / CARDINALITY)
+NUMBER_OF_DUPLICATE = int((NUMBER_OF_SUBSETS-CARDINALITY)/(CARDINALITY/2))
+UNCOVERED_ELEMENTS = set(ELEMENTS_OF_UNIVERSE)
+TEST_LOOP = 1
+
+
+def making_universe(SIZE_OF_UNIVERSE, CARDINALITY):
```

Comment:
methods are more commonly "actions" so this wuold be better called `make_universe(...)`. also, the `CAPITAL_NOTATION` is reserved for constants, like you put at the top of the file. parameters use `underscore_naming`. Pycharm should've warned you about this. **be aware of the style warnings  pycharm gives you!**